src/app/qtlocalpeer/qtlockedfile_win.cpp

```diff
@@ -89,15 +89,15 @@ Qt::HANDLE QtLockedFile::getMutexHandle(int idx, bool doCreate)
 
     Qt::HANDLE mutex;
     if (doCreate) {
-        QT_WA( { mutex = CreateMutexW(NULL, FALSE, (TCHAR*)mname.utf16()); },
+        QT_WA( { mutex = CreateMutexW(NULL, FALSE, reinterpret_cast<const TCHAR *>(mname.utf16())); },
```

Comment:
Seems this file doesn't still follow qBittorrent's one.

Reply:
What do you mean exactly?

Reply:
>What do you mean exactly?

There are a lot of other coding style violations here. Wouldn't it be better to fix them all?

Reply:
I only intend to fix the compilation warnings I encountered.

Reply:
>I only intend to fix the compilation warnings I encountered.

Ok.